cfg/ck8s-cis-1.24/node.yaml

```diff
@@ -0,0 +1,476 @@
+            - flag: --streaming-connection-idle-timeout
+              path: '{.streamingConnectionIdleTimeout}'
+              set: false
+          bin_op: or
+        remediation: |
+          Edit the kubelet configuration file
+          $kubeletconf on each worker node and set the following argument.
+
+          `--streaming-connection-idle-timeout=5m`
+
+          Restart the kubelet service.
+
+          For example, `snap restart k8s.kubelet`
+        scored: false
+
+      - id: 4.2.6
+        text: "Ensure that the --protect-kernel-defaults argument is set to true (Automated)"
+        audit: "/bin/ps -fC $kubeletbin"
+        audit_config: "/bin/cat $kubeletconf"
+        tests:
+          test_items:
+            - flag: --protect-kernel-defaults
+              path: '{.protectKernelDefaults}'
+              compare:
+                op: eq
+                value: true
+        remediation: |
+          Edit the kubelet configuration file
+          $kubeletconf on each worker node and set the following argument.
+
+          `--protect-kernel-defaults=true`
+
+          Restart the kubelet service.
+
+          For example, `snap restart k8s.kubelet`
+        scored: true
+
+      - id: 4.2.7
+        text: "Ensure that the --make-iptables-util-chains argument is set to true (Automated)"
+        audit: "/bin/ps -fC $kubeletbin"
+        audit_config: "/bin/cat $kubeletconf"
+        tests:
+          test_items:
+            - flag: --make-iptables-util-chains
+              path: '{.makeIPTablesUtilChains}'
+              compare:
+                op: eq
+                value: true
+            - flag: --make-iptables-util-chains
+              path: '{.makeIPTablesUtilChains}'
+              set: false
+          bin_op: or
+        remediation: |
+          Edit the kubelet configuration file
+          $kubeletconf on each worker node and
+          remove the --make-iptables-util-chains argument.
+
+          Restart the kubelet service.
+
+          For example: `snap restart k8s.kubelet`
+        scored: true
+
+      - id: 4.2.8
+        text: "Ensure that the --hostname-override argument is not set (Manual)"
+        # This is one of those properties that can only be set as a command line argument.
+        # To check if the property is set as expected, we need to parse the kubelet command
+        # instead reading the Kubelet Configuration file.
+        audit: "/bin/ps -fC $kubeletbin "
+        tests:
+          test_items:
+            - flag: --hostname-override
+              set: false
+        remediation: |
+          Edit the kubelet configuration file $kubeletconf
+          on each worker node and remove the --hostname-override argument.
+
+          Restart the kubelet service.
+
+          For example, `snap restart k8s.kubelet`
+        scored: false
+
+      - id: 4.2.9
+        text: "Ensure that the eventRecordQPS argument is set to a level which ensures appropriate event capture (Manual)"
```

Comment:
nit: all other texts use the flag

Ensure that the `--event-qps` argument is set to a level that ensures appropriate event capture (Manual).